### DISCUSSION TITLE
How do I customise the border around DropDown element?

### DISCUSSION THREAD

**Discussion by lalavany-microsoft-com** [microsoft/fluentui#18831] [Q&A] [answered]
What I need is: 
<img width="72" alt="Screenshot 2021-07-04 at 11 42 21 PM" src="https://user-images.githubusercontent.com/44528885/124395397-9053c380-dd21-11eb-8d9d-8c09f632b877.png">

But what I have after implementing onRenderTitle & onRenderCaretDown is this: 
<img width="99" alt="Screenshot 2021-07-04 at 11 42 00 PM" src="https://user-images.githubusercontent.com/44528885/124395417-afeaec00-dd21-11eb-9ae8-115b4657ec09.png">

Is there a way to get rid of the border around the DropDown component?

**Comment by khmakoto** [comment]
@lalavany-microsoft-com do you have a codepen I could look and fiddle with? Thanks!

**Reply by lalavany-microsoft-com** [reply]
@khmakoto Thank you for responding.
Check below link to fiddle with
https://codepen.io/lalavany/pen/yLbOzxY

**Comment by khmakoto** [comment, marked as answer]
@lalavany-microsoft-com [here](https://codepen.io/khmakoto/pen/ExmywRy?editors=0010) is a modified codepen with no border around the `Dropdown`, hopefully this helps you!

**Reply by lalavany-microsoft-com** [reply]
yay.. thank you for this @khmakoto .
Can you also tell me how to get rid of the blue border which shows up after making a selection from drop down options?

<img width="72" alt="Screenshot 2021-07-09 at 2 41 49 PM" src="https://user-images.githubusercontent.com/44528885/125054578-ff377080-e0c3-11eb-9028-9bc304fdd207.png">

**Reply by khmakoto** [reply]
Hey, for sure, I have further modified the codepen to remove that as well. Let me know if there is anything else I can do to help.

**Reply by lalavany-microsoft-com** [reply]
Nice.. thank you so much for helping me through this :)

Really appreciate it.

**Reply by khmakoto** [reply]
You are very welcome!
